lemur/models.py

```diff
@@ -49,20 +43,16 @@ class Lab(db.Model):
     id = db.Column(db.String(128), nullable=False, unique=True,
                    primary_key=True)
     name = db.Column(db.String(64), nullable=False)
-    class_name = db.Column(db.String(32), nullable=False)
-    prof_name = db.Column(db.String(32), nullable=False)
     # lab description contained in description variable
     description = db.Column(db.String(4096))
     status = db.Column(db.Enum('Activated', 'Downloadable', 'Unactivated',
                                name='status'))
-
     # one-to-many: a lab can have multiple experiments
     experiments = db.relationship('Experiment', back_populates='lab',
                                   cascade="all, delete, delete-orphan")
-    # Many-to-Many: a class can have multiple labs and a lab can have multiple
-    # classes
-    classes = db.relationship("Class", secondary=association_table_class_lab,
-                              back_populates="labs")
+    # Many-to-One: a class can have multiple labs
+    class_id = db.Column(db.String(128), db.ForeignKey('Class.id'))
```

Comment:
Is the class id intended to be user input? Are there any guarantees about it apart from that it is a string?